### ISSUE TITLE
Transport dupe in the Guild region

### ISSUE THREAD

**Issue by MaxIsV** [guilds-plugin/Guilds#348]
## Description

When you place any transport from the "Vehicles" plugin while in the Guild region, a transport dupe will happen. Then you can pick up the transport and break the chest, then you will get 2 vehicles, which are fully working. For a dupe to work, you must be added to the Guild region.

## Expected Behavior

The transport must be placed, and the chest must disappear.

## Actual Behavior

Transport is placed, but the chest does not disappear, and it is also placed.

## Steps to Reproduce

Video: https://imgur.com/a/1dNGTq3

## Your Environment

* Guilds version 3.5.5.2
* MC Version Paper-1.12.2
* Plugin List WorldGuard, Vehicles...

**Comment by darbyjack** [comment]
I don't really know what Vehicles is, and I'm not really sure how to fix that. Sounds more like a WorldGuard-related problem with Vehicles, since that is what Guilds currently uses for claims.

**Comment by MaxIsV** [comment]
> I don't really know what Vehicles is, and I'm not really sure how to fix that. Sounds more like a WorldGuard-related problem with Vehicles, since that is what Guilds currently uses for claims.

This bug occurs only in the regions of the Guild. In all other WorldGuard regions, this does not occur.

**Comment by darbyjack** [comment]
Can you link me the plugin for Vehicles? The only one I'm seeing on SpigotMC is premium, and I can't do anything about that if I can't test it.

**Comment by MaxIsV** [comment]
> Can you link me the plugin for Vehicles? The only one I'm seeing on SpigotMC is premium, and I can't do anything about that if I can't test it.

Yes, this is a premium plugin.  But I do not use its latest version, as it does not support 1.12.2.  Do you want me to give you a link to download the plugin?

**Comment by darbyjack** [comment]
If it is public, sure, I will not touch a leak of a premium plugin.

**Comment by MaxIsV** [comment]
> If it is public, sure, I will not touch a leak of a premium plugin.

I do not quite understand what you mean by "public".  But if I own a premium plugin on Spigot, what is bad if I send it to you to add compatibility?  After all, dupe is important.

**Comment by darbyjack** [comment]
It is still considered leaking. I will try and reach out to the developer of the project myself.

**Comment by darbyjack** [comment]
Update: I've found his Discord, I am in the works of getting in contact with him.

**Comment by MaxIsV** [comment]
@darbyjack So... is there any news on this?

**Comment by darbyjack** [comment]
> @darbyjack So... is there any news on this?

I apologize but I have not had time to do much in terms of fixing this. Real life has taken over and it's not really allowing me much time to take the time to fix this. If someone else would like to PR a fix for it, that's more than welcomed. I plan to be dropping WorldGuard claims here in the future when the new claim system is done so I think that will fix the bug itself.
